### PR TITLE
Change snmp configuration setup to a command.

### PR DESCRIPTION
**Pull request by stephen-soltesz**
This change eliminates the collectd-setup init script in favor of a stand-alone command.

The stand-alone command is called automatically during collectd start.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/collectd-mlab/49)

### CODE REVIEW

File: collectd-mlab.spec [hunk 1]
```diff
@@ -150,7 +150,7 @@ rm -rf $RPM_BUILD_ROOT
 /usr/share/collectd-mlab/types.db
 
 # Init script.
```
Comment (nkinkade) [inline collectd-mlab.spec]: Is this comment still valid?

File: collectd-mlab.spec [hunk 2]
```diff
@@ -150,7 +150,7 @@ rm -rf $RPM_BUILD_ROOT
...
-/etc/init.d/collectd-setup
+/etc/init.d/collectd_snmp_setup.sh
```
Comment (nkinkade) [inline collectd-mlab.spec]: Should this be _/usr/bin/collectd_snmp_setup.sh_?

File: collectd-mlab.spec (hunk 1 above)
Comment (stephen-soltesz) [inline collectd-mlab.spec]: No this was a mistake. Fixed.

File: collectd-mlab.spec (hunk 2 above)
Comment (stephen-soltesz) [inline collectd-mlab.spec]: Uuf. Also a mistake. Fixed. Thank you for catching this.

**Comment by nkinkade** [comment]
LGTM

**Pull request merged**